app/frontend/src/components/stories/Avatar.stories.tsx

```diff
@@ -0,0 +1,40 @@
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
```

Comment:
What breaking changes? Maybe we'd better do that since we don't have much in yet?

Reply:
Ok, I will do that in another PR